exopy/measurement/processor.py

```diff
@@ -311,10 +317,11 @@ def _run_measurement(self, measurement):
         errors = {}
         if self._check_for_pause_or_stop():
 
-            # Connect new monitors, and start them.
-            logger.debug('Connecting monitors for measurement %s',
-                         meas_id)
-            self._start_monitors(measurement)
+            if not headless:
```

Comment:
Non-UI related monitors could exist (email senders, slack, etc) I would prefer to add to monitor a field declaring whether or not they rely o the GUI and delegate the decision to start things to `_start_monitors`.

Reply:
AFAIK these monitors don't exist/aren't used and this headless mode is for automatic calibrations where we don't want any monitors at all since this would spam all those non GUI monitors very quickly.
If you feel strongly about this, I can change it, but for now, I think that headless should be associated with quick, automatic, somewhat off the record measurements (we only want the measurement results at the end).